jsk_rviz_plugins/src/pie_chart_display.cpp

```diff
@@ -359,6 +363,11 @@ namespace jsk_rviz_plugins
     
   }
   
+  void PieChartDisplay::updateCaption()
+  {
+    caption_ = caption_property_->getStdString();
```

Comment:
Why not below implementation for backward compatibility of current existing rviz config file, and convenience.
```c++
std::string caption =  caption_property_->getStdString();
if (caption.empty()) {
  caption_ = std::string(getName());  // the name of display
} else {
  caption_ = caption;
}
```